common/helpers/picasso.js

```diff
@@ -0,0 +1,216 @@
+import minBy from 'lodash/minBy'
+import maxBy from 'lodash/maxBy'
+import sortBy from 'lodash/sortBy'
+
+/**
+ * Splits a set of days into pages, such that all pages contain a similar number of days.
+ *
+ * @param days {array} set of days to split into pages
+ * @param maxDaysPerPage {number} maximum number of days to put on one page
+ * @returns {array} list of pages, each containing a list of the days on the page
+ */
+export function splitDaysIntoPages(days, maxDaysPerPage) {
+  const numberOfDays = days.length
+  const numberOfPages = Math.ceil(numberOfDays / maxDaysPerPage)
+  const daysPerPage = Math.floor(numberOfDays / numberOfPages)
+  const numLargerPages = numberOfDays % numberOfPages
+  let nextUnassignedDayIndex = 0
+  if (isNaN(numberOfPages)) return []
```

Comment:
For which case is this condition needed?
You could move the argument validation to the top of the method

Reply:
It's only a very special case when maxDaysPerPage is 0. I moved the condition to the top and made it more explicit in [3018799](https://github.com/ecamp/ecamp3/pull/3507/commits/3018799be97f744fc434e1a37c18a64830c689bd)